packages/vscode-extension/lib/babel_transformer.js

```diff
@@ -111,19 +111,21 @@ function transformWrapper({ filename, src, ...rest }) {
     ) {
       src = `module.exports = require("__RNIDE_lib__/rn-renderer/react-native-74-77/${rendererFileName}");`;
     }
-    if (version.startsWith("0.78")) {
+    if (version.startsWith("0.78") || version.startsWith("0.79")) {
       src = `module.exports = require("__RNIDE_lib__/rn-renderer/react-native-78/${rendererFileName}");`;
     }
-  } else if (
-    isTransforming("node_modules/react/cjs/react-jsx-dev-runtime.development.js")
-  ) {
+  } else if (isTransforming("node_modules/react/cjs/react-jsx-dev-runtime.development.js")) {
     const { version } = requireFromAppDir("react-native/package.json");
     const jsxRuntimeFileName = filename.split(path.sep).pop();
-    if (version.startsWith("0.78")) {
+    if (version.startsWith("0.78") || version.startsWith("0.79")) {
       src = `module.exports = require("__RNIDE_lib__/JSXRuntime/react-native-78/${jsxRuntimeFileName}");`;
```

Comment:
You should rename the directory to
```suggestion
      src = `module.exports = require("__RNIDE_lib__/JSXRuntime/react-native-78-79/${jsxRuntimeFileName}");`;
```
or similar